governance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Governance 1.0 DRAFT WIP
+description: Governance 1.0 discussion draft
+---
+# Governance 1.0 `DRAFT WIP`
+
+This document contains a work in progress (WIP) draft of the initial governance of the Op1st project (Governance 1.0).
+
+It also has a roadmap for additions coming in the next iteration of governance (Governane 1.1).
+
+## Op1st Community Governance 1.0 (MVG)
+
+A healthy open source project is one that demonstrates open practices, uses open infrastructure, and cultivates an open culture with the goal of becoming more sustainable.
+
+### Principles
+As a new cloud-native community growing up at the footsteps of massively scaled and successful projects such as Kubernetes and OpenInfra, we want to be careful to learn from those who came before us, while forging our own path into the challenges we've chosen to undertake.
+
+To stand up this Governance for a 1.0 good-enough minimal-viable-governance (MVG), we are looking to our key predecessors for some of our starting points.
+As an initial set of principles we are combining elements of the [Kubernetes community principles](https://github.com/kubernetes/community/blob/master/governance.md#principles) and the [OpenInfra Foundations Four Opens](https://openinfra.dev/four-opens/).
+
+#### Op1st Community Principles 1.0
+Our mission requires us to use open source principles in ways beyond licensing the code we write, but at the core we are an open source community and want to adhere to the best possible principles for creating an open, welcoming, and inclusive community.
+
+These are the Op1st community principles we adhere to:
+1. **Open Source** : Any content we create is released under an open source license. Content includes software, documentations, configuration files, and training materials.
```

Comment:
Should we specify/determine the license here itself?

Reply:
👎 the license should not be enforced.

Reply:
@quaid please add `operational data` to the list of content. Or just `data`

Reply:
Similar to @durandom 's comment, since this is about operate first, should we add add "operational code" or "code necessary to operate software".

Reply:
Content could be documentation. Should the sentence then say 'open source or open content license'?

Reply:
```suggestion
1. **Open Source** : Any content we create is released under an open work license. Content includes all software, documentations, configuration files, data, and training materials.
"All software" includes the code necessary to operate software, AKA _operational code_. 
```

Reply:
@durandom @stefwalter I think this covers your cases?

@pilhuhn very good point, and because we have other output such as data, AI/ML artifacts, and artwork (logos, diagrams, etc.), I changed to the completely-inclusive "open work" such as how [Open@RIT uses it](https://drive.google.com/file/d/1DjjuEnfJKxfDFdktDoc_3Xx2ne8xp9fX/view).

Reply:
yes, full ACK. This looks good

Reply:
@quaid Works for me. Although I would have liked to see operational code clearly called out, since this is often ignored, I understand that it's covered by the principle.

Reply:
> @quaid Works for me. Although I would have liked to see operational code clearly called out, since this is often ignored, I understand that it's covered by the principle.

You know what, you've convinced me—when working on mindset changes, it helps to highlight these definitions that might not be assumptive to everyone yet. I'll work this in a way that benefits the cause.